.pipelines/java_packaging.yml

```diff
@@ -13,6 +13,23 @@ stages:
     pool: {name: 'onnxruntime-Win-CPU-2022'}
 
     steps:
+    - task: PowerShell@2
+      displayName: 'Set version'
+      inputs:
+        targetType: 'inline'
+        script: |
+          $_ExtVersion=(cat version.txt)
+          echo "##vso[task.setvariable variable=ExtVersion;]$_ExtVersion"
+        workingDirectory: '$(Build.SourcesDirectory)'
+
+    - script: |
+        echo "##vso[task.setvariable variable=OrtExtVersion;isOutput=true]$(ExtVersion)"
```

Comment:
why do we need 2 setvariable here?

Reply:
tried with one - but output variables (with isOutput=true) were not working in a PowerShell script, and we need output variables to be able to access them from the other jobs, so this was the workaround.

Reply:
I just ask ChatGPT about how to do that on batch file, it gives a pretty neat solution.

set /p FILE_CONTENT=<example.txt
echo %FILE_CONTENT%

use ADO setvariable looks like a little longsome.